src/codeu/chat/client/simplegui/ConversationPanel.java

```diff
@@ -134,6 +137,26 @@ public void actionPerformed(ActionEvent e) {
       }
     });
 
+    // User clicks Conversations Rename button.
+    renameButton.addActionListener(new ActionListener() {
```

Comment:
nit: it's nice to use the newest Java lambda syntax:
renameButton.addActionListener((ActionEvent e) -> {
if (objectList.getSelectedIndex()....
});

https://www.geeksforgeeks.org/lambda-expressions-java-8/

Reply:
wut... i expected a change :|